### ISSUE TITLE
GUIなGitクライアントの紹介を書きたい

### ISSUE THREAD

**Issue by meokz** [gitpractice-enpit/github_practice#14]
・Source  Tree
・GitKraken

**Comment by urushiyama** [comment]
GitKrakenいいかな、と思って使ってみましたが、
- ライセンスが非商用利用に限るものなのでPrivateリポジトリを開くたびに「商用利用ならPro使えよ」と警告が出る
- 起動が比較的遅い
- バイナリファイルへの対応がSourceTreeに比べて弱い
- SourceTreeは過去にコミットした状態のファイルをcheckoutせずに開けるが、GitKrakenではcheckoutするほかなく、しかし特定のcommitにcheckoutするメニューもみた感じない
- UIが全体的に大きめで画面の解像度によってはグラフに横スクロールが発生する
- UIのレイアウト変更の自由度が低い

というところが気になります。
一方でGitKrakenいいな、と思った点は、
- ブランチの表示がコミットメッセージと独立していてみやすい
- グラフ見ただけで誰のコミットかわかる
- コミットメッセージが書きやすい
- Windows, Mac, Linuxで環境を揃えられる
- Githubのアカウントだけで利用可能

といったところですね。

**Comment by meokz** [comment]
@urushiyama 

Githubに学生登録すると幸せになるかもしれません
> ライセンスが非商用利用に限るものなのでPrivateリポジトリを開くたびに「商用利用ならPro使えよ」と警告が出る

特定のcommitにcheckoutできますよ！そのcommitを右クリックで出ます！
> SourceTreeは過去にコミットした状態のファイルをcheckoutせずに開けるが、GitKrakenではcheckoutするほかなく、しかし特定のcommitにcheckoutするメニューもみた感じない

**Comment by urushiyama** [comment]
@meokz 

> Githubに学生登録すると幸せになるかもしれません

いや、学生登録はしてます。その上で、GitHubのPrivateリポジトリを開くとGitKraken上にアラートが出るんです。

> 特定のcommitにcheckoutできますよ！そのcommitを右クリックで出ます！

手元のmac版GitKraken version2.6.0ではbranchへのチェックアウトはできるんですが、commitには右クリックでもcheckoutできないです。[stack overflowにもこの話題上がってました。](https://stackoverflow.com/questions/39977015/how-to-checkout-old-commit-in-git-kraken)
